e2e/tests/manageCases/caseProgression/completeTheOrder/completeTheOrder.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { Page, test } from "@playwright/test";
+import Config from "../../../../config";
+import createDaCitizenCourtNavCase from "../../../../common/createCaseHelper";
+import { Helpers } from "../../../../common/helpers";
+import config from "../../../../config";
+import { CompleteTheOrder } from "../../../../journeys/manageCases/caseProgression/completeTheOrder/completeTheOrder";
+
+test.use({ storageState: Config.sessionStoragePath + "caseWorker.json" });
+
+test.describe("Complete the Order task for DA Citizen case tests.", () => {
+  let ccdRef: string = "";
+
+  test.beforeEach(async ({ page }) => {
+    ccdRef = await createDaCitizenCourtNavCase(true, false);
+    await Helpers.goToCase(page, config.manageCasesBaseURL, ccdRef, "tasks");
+  });
+
+  test("Complete Complete The Order without accessibility test. @regression", async ({
+    page,
+    browser,
+  }): Promise<void> => {
+    await CompleteTheOrder.completeTheOrder({
+      page: page,
+      accessibilityTest: false,
+      yesNoSendToGateKeeper: true,
+      ccdRef: ccdRef,
+      c100CaseWorkerActions: "Manage orders",
+      manageOrdersOptions: "create order",
+      createOrderFL401Options: "power of arrest",
+      yesNoManageOrders: false,
+      judgeTitles: "Her Honour Judge",
+      withOrWithoutNotice: true,
+      createOrderManageOrders19Options: "dateToBeFixed", // "dateConfirmed" will not pass because page 19 does not give a hearing you are allowed to select
+      howLongWillOrderBeInForce: "untilNextHearing", // Should not matter unless non-molestation order is selected.
+      browser: browser,
+    });
+  });
+
+  test("Complete Complete The Order with accessibility test. @regression @accessibility", async ({
+    page,
+    browser,
+  }): Promise<void> => {
+    await CompleteTheOrder.completeTheOrder({
+      page: page,
+      accessibilityTest: false,
+      yesNoSendToGateKeeper: true,
+      ccdRef: ccdRef,
+      c100CaseWorkerActions: "Manage orders",
+      manageOrdersOptions: "create order",
+      createOrderFL401Options: "power of arrest",
+      yesNoManageOrders: false,
+      judgeTitles: "Her Honour Judge",
```

Comment:
and this to this:         judgeTitles: "Deputy Circuit Judge",

Just to separate the tests as they are both the same test ultimately. The arguments are accounted for in judge submit page too